src/rollup/types.d.ts

```diff
@@ -19,7 +19,7 @@ export type RollupWarning = RollupLog;
 
 export interface RollupLog {
 	binding?: string;
-	cause?: Error;
+	cause?: Error | unknown;
```

Comment:
Just change it to `unknown`. `Error | unknown` is equivalent, just more complicated.

Reply:
Okay I can do this, but as the tests are failing I need to spin up my dev environment and look into it what needs to be fixed